docs/admin/cluster-components.md

```diff
@@ -15,7 +15,7 @@ cluster (e.g., scheduling), and detecting and responding to cluster events
 (e.g., starting up a new pod when a replication controller's 'replicas' field is
 unsatisfied).
 
-Master components could in theory be run on any node in the cluster. However,
+Master components could in theory be running on any node in the cluster. However,
```

Comment:
I would suggest: "**In theory,** Master components **can** be **run** on any node in the cluster."